js/modal.js

```diff
@@ -64,6 +66,28 @@
     this.resize()
 
     this.$element.on('click.dismiss.bs.modal', '[data-dismiss="modal"]', $.proxy(this.hide, this))
+    this.$element.on('show.bs.collapse hide.bs.collapse', function (e) {
```

Comment:
i feel like this is making a bloated method, worse – can we move some of this logic into a handler

maybe like 

```js
this.$element.on('show.bs.collapse hide.bs.collapse', this.adjustDialogOnShowHide)
```
or some better name :+1: